### PR TITLE
fix: donation clean up

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/components/Donation/donate-form.tsx [hunk 1]
```diff
@@ -26,14 +26,14 @@ import {
...
-import DonateCompletion from './DonateCompletion';
+import DonateCompletion from './donate-completion';
 import PatreonButton from './patreon-button';
 import type { AddDonationData } from './paypal-button';
 import PaypalButton from './paypal-button';
 import StripeCardForm, { HandleAuthentication } from './stripe-card-form';
 import WalletsWrapper from './walletsButton';
 
-import './Donation.css';
+import './donation.css';
```
Comment (nhcarrigan) [inline client/src/components/Donation/donate-form.tsx:36]: Looks like either the file needs to be renamed or this change needs to be reverted.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: client/src/components/Donation/donate-form.tsx (hunk 1 above)
Comment (ahmadabdolsaheb) [reply client/src/components/Donation/donate-form.tsx:36]: I renamed that file, but git somehow did not pick it up. :)

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM :+1:

**Comment by ahmadabdolsaheb** [comment]
@nhcarrigan, how does it look after the changes?

**Review by ojeytonwilliams — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
This is looking good to me.

**Pull request merged**